app/src/pages/DesignSystem.svelte

```diff
@@ -55,284 +57,379 @@
-      </td>
-      <td>
-        <Button size="small" icon={Icon.SearchSmall} variant="secondary">
-          Secondary
-        </Button>
-      </td>
-      <td>
-        <Button
-          size="small"
-          icon={Icon.SearchSmall}
-          variant="secondary"
-          disabled>
-          Secondary
-        </Button>
-      </td>
-    </tr>
+        <tr>
+          <td>
+            <Button size="small" variant="outline">Outline</Button>
+          </td>
+          <td>
+            <Button size="small" variant="outline" disabled>Outline</Button>
+          </td>
+          <td>
+            <Button size="small" icon={Icon.Graph} variant="outline">
+              Outline
+            </Button>
+          </td>
+          <td>
+            <Button size="small" icon={Icon.Graph} variant="outline" disabled>
+              Outline
+            </Button>
+          </td>
+        </tr>
+      </table>
+    </div>
+  </div>
 
-    <tr>
-      <td>
-        <Button size="small" variant="transparent">Transparent</Button>
-      </td>
-      <td>
-        <Button size="small" variant="transparent" disabled>Transparent</Button>
-      </td>
-      <td>
-        <Button size="small" icon={Icon.Check} variant="transparent">
-          Transparent
-        </Button>
-      </td>
-      <td>
-        <Button size="small" icon={Icon.Check} variant="transparent" disabled>
-          Transparent
-        </Button>
-      </td>
-    </tr>
+  <div class="section">
+    <div class="first-column">
+      <Title.Big>Form elements</Title.Big>
+      <Text.Regular style="color: var(--color-gray)">
+        Inputs, text areas, dropdowns, etc.
+      </Text.Regular>
+    </div>
+    <div class="second-column">
+      <div class="row">
+        <Input placeholder="Hey, I'm an input." />
+      </div>
 
-    <tr>
-      <td>
-        <Button size="small" variant="outline">Outline</Button>
-      </td>
-      <td>
-        <Button size="small" variant="outline" disabled>Outline</Button>
-      </td>
-      <td>
-        <Button size="small" icon={Icon.Graph} variant="outline">
-          Outline
-        </Button>
-      </td>
-      <td>
-        <Button size="small" icon={Icon.Graph} variant="outline" disabled>
-          Outline
-        </Button>
-      </td>
-    </tr>
-  </table>
+      <div class="row">
+        <Select items={['master', 'dev']} value={'dev'} />
+        <Select disabled items={['master', 'dev']} value={'dev'} />
+      </div>
 
-  <div>
-    <Input placeholder="Hey, I'm an input." />
+      <div class="row">
+        <CheckboxInput>How about a checkbox?</CheckboxInput>
+      </div>
+    </div>
   </div>
 
-  <div>
-    <Select items={['master', 'dev']} value={'dev'} />
+  <div class="section">
+    <div class="first-column">
+      <Title.Big>Components</Title.Big>
+      <Text.Regular style="color: var(--color-gray)">
+        More complex UI elements
+      </Text.Regular>
+    </div>
+    <div class="second-column">
```

Comment:
Can we add a heading or some similar text to show that this is a notifiction we are looking at?